src/actions/WalletActions.tsx

```diff
@@ -21,6 +26,104 @@ export interface SelectWalletTokenParams {
   alwaysActivate?: boolean
 }
 
+const activateWalletName: MapObject<{ name: string; notes: string }> = {
+  ripple: {
+    name: s.strings.activate_wallet_token_transaction_name_xrp,
+    notes: s.strings.activate_wallet_token_transaction_notes_xrp
+  }
+}
+
+const ACTIVATION_TOAST_AUTO_HIDE_MS = 5000
+
+const activateWalletTokens = async (
```

Comment:
We don't need to fix this, but the Edge convention is to put un-exported helper functions at the bottom of the file.